projects/plugins/social/src/js/components/admin-page/index.jsx

```diff
@@ -68,11 +69,17 @@ const Admin = () => {
 		select => select( CONNECTION_STORE_ID ).getConnectionStatus(),
 		[]
 	);
+	const { jetpackSocialConnectionsAdminUrl } = window.jetpackSocialInitialState;
 	const { isUserConnected, isRegistered } = connectionStatus;
 	const showConnectionCard = ! isRegistered || ! isUserConnected;
 	return (
 		<AdminPage moduleName={ __( 'Jetpack Social', 'jetpack-social' ) }>
 			<AdminSectionHero>
+				<div className={ styles.manageConnectionsHeader }>
```

Comment:
I think we should move this to within the check for `showConnectionCard` below, as we only want it to display when we have a Jetpack connection and the ability to manage the Publicize connections